### PR TITLE
Update Synapse ARM template

### PR DESCRIPTION
**Pull request by coolswatish**

### CODE REVIEW

File: Manage/DeployWorkspace/azuredeploy.json [hunk 1]
```diff
@@ -2,23 +2,29 @@
...
+            "type": "String",
+            "metadata": {
+                "description": "Provide the user name for SQL server login."
+            }
         },
         "sqlAdministratorPassword": {
-            "type": "SecureString"
+            "type": "SecureString",
+            "metadata": {
+                "description": "The password for the SQL server. Visit https://aka.ms/azuresqlserverpasswordpolicy for password policy."
```
Comment (jovanpop-msft) [inline Manage/DeployWorkspace/azuredeploy.json:26]: Change it to the password for the serverless SQL pool endpoint. there is no SQL Server in Synapse.

File: Manage/DeployWorkspace/azuredeploy.json [hunk 2]
```diff
@@ -2,23 +2,29 @@
...
             "metadata": {
-                "description": "The name of the storage. If not provided, the workspace name will be used."
+                "description": "The name of the primary ADLS Gen2 Storage Account. If not provided, the workspace name will be used."
             }
         },
         "sqlAdministratorLogin": {
-            "type": "String"
+            "type": "String",
+            "metadata": {
+                "description": "Provide the user name for SQL server login."
```
Comment (jovanpop-msft) [inline Manage/DeployWorkspace/azuredeploy.json:20]: for serverless SQL pool login

Comment (coolswatish) [reply Manage/DeployWorkspace/azuredeploy.json]: Updated this to "SQL login"

File: Manage/DeployWorkspace/azuredeploy.json (hunk 1 above)
Comment (coolswatish) [reply Manage/DeployWorkspace/azuredeploy.json]: Updated this.

**Pull request merged**